src/Spreadsheet.js

```diff
@@ -115,12 +114,6 @@ class Spreadsheet<CellType, Value> extends PureComponent<{|
 
   formulaParser = new FormulaParser();
 
-  /**
-   * Internally used value to check if the copied text match the live objects
-   * inside state.copied
-   */
-  _clippedText: string | null = null;
```

Comment:
Why did you delete this?

Reply:
Since it always reads the the text from the clipboard, I figured this is not needed anymore.